controller/internal/routes/current_api_session_api_model.go

```diff
@@ -90,6 +90,9 @@ func MapApiSessionAuthQueriesToRestEntity(ae *env.AppEnv, rc *response.RequestCo
 			MinLength:  authQuery.MinLength,
 			Provider:   authQuery.Provider,
 			TypeID:     authQuery.TypeID,
```

Comment:
I don't see these fields populated in API session returned from `/authenticate?method=cert` call

Reply:
That would be odd for legacy, specifically in this PR, because there are tests in this PR that ensure that it is the exact thing.

Reply:
I also added a set of OIDC value checks, and the values I put in seem to come out.